src/ReadVTK.jl

```diff
@@ -126,16 +128,27 @@ function VTKFile(filename)
   # Ensure supported compression type
   @assert in(compressor, ("", "vtkZLibDataCompressor"))
 
-  # Extract piece
-  piece = root[file_type][1]["Piece"][1]
-  n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
-  n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+  if file_type == "UnstructuredGrid"
+    # Extract piece
+    piece = root[file_type][1]["Piece"][1]
+    n_points = parse(Int, attribute(piece, "NumberOfPoints", required=true))
+    n_cells = parse(Int, attribute(piece, "NumberOfCells", required=true))
+
+  elseif file_type == "ImageData"
+    piece = root[file_type][1]
+    wholeExtent = parse.(Int,     split( attribute(piece, "WholeExtent", required=true) , ' ' ) )
+    n_points_per_grid_dir = [ wholeExtent[2*i]+1 for i in (1:3) ]
+    n_points = prod( n_points_per_grid_dir )
+    n_cells = prod( n_points_per_grid_dir .- 1 )
```

Comment:
```suggestion
    n_points_per_grid_dir = [wholeExtent[2*i]+1 for i in (1:3)]
    n_points = prod(n_points_per_grid_dir)
    n_cells = prod(n_points_per_grid_dir .- 1)
```
We generally do not use a space inside parens/brackets. Would you mind changing this also elsewhere to keep the formatting style consistent with the existing code? It seems trivial, but in our experience it greatly helps in reading code written by someone else if everything is formatted consistently.